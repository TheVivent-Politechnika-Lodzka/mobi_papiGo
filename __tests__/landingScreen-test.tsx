import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import LandingScreen from '../src/screens/auth/landingScreen';

it('Button render test', () => {
  const land = renderer.create(<LandingScreen />).toJSON();
  expect(land).toMatchSnapshot();
});
