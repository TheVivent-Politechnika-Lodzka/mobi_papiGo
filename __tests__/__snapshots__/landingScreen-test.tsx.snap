// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#87AAAA",
      "bottom": 0,
      "flex": 1,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(255,255,255,0)",
        "flex": 4,
        "height": "60%",
        "width": "80%",
      }
    }
  >
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Image
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../assets/logo.png",
          }
        }
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": undefined,
              "width": undefined,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "rgba(255,255,255,0)",
        "marginBottom": 100,
        "marginHorizontal": "auto",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 3,
            },
            Object {
              "marginHorizontal": 50,
              "marginVertical": 10,
              "width": 200,
            },
            false,
          ]
        }
      >
        <View
          accessibilityRole="button"
          accessibilityState={
            Object {
              "busy": false,
              "disabled": false,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          nativeID="animatedComponent"
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#C8E3D4",
                "borderColor": "white",
                "borderRadius": 30,
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 8,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 18,
                  "fontWeight": "bold",
                  "paddingVertical": 1,
                  "textAlign": "center",
                }
              }
            >
              Logowanie
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "borderRadius": 3,
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 3,
            },
            Object {
              "marginHorizontal": 50,
              "marginVertical": 10,
              "width": 200,
            },
            false,
          ]
        }
      >
        <View
          accessibilityRole="button"
          accessibilityState={
            Object {
              "busy": false,
              "disabled": false,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          nativeID="animatedComponent"
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#C8E3D4",
                "borderColor": "white",
                "borderRadius": 30,
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "marginBottom": "10%",
                "padding": 8,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000",
                  "fontSize": 18,
                  "fontWeight": "bold",
                  "paddingVertical": 1,
                  "textAlign": "center",
                }
              }
            >
              Rejestracja
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
